common/src/OpenSpaceNet.cpp

```diff
@@ -459,31 +463,17 @@ void OpenSpaceNet::processSerial()
     skipLine();
     OSN_LOG(info) << "Reading image...";
```

Comment:
It is confusing that you have "Reading image" and then (for the detect action) "Reading" is a progress bar.  Perhaps "Loading image.." and then the progress bar always shows "Reading".

(See the long comment below for details)